go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/HdrHistogram/hdrhistogram-go v1.1.2 // indirect
 	github.com/apache/thrift v0.13.0
 	github.com/bytedance/gopkg v0.0.0-20210910103821-e4efae9c17c3
+	github.com/choleraehyq/pid v0.0.13 // indirect
 	github.com/cloudwego/kitex v0.1.4
```

Comment:
could you please upgrade kitex to v0.2.1 and remove the dependency github.com/choleraehyq/pid

Reply:
@YangruiEmma Thanks, I just updated the code, now the kitex version is already v0.2.1, and github.com/choleraehyq/pid was removed. I tested it locally, it works well.